tests/integration/Makefile

```diff
@@ -30,52 +30,75 @@ endif
 
 .PHONY : all
 all:
```

Comment:
Why running command instead of declaring them as dependency?

```make
all: client_endpoints dynamic_record s_client s_server gnutls_cli gnutls_serv sslyze
```

Reply:
thanks for the suggestion - I realize I tried declaring them on multiple lines which didn't work, will update this PR to just this